nservicebus/messaging/databus/index_obsolete_core_[9.1,).partial.md

```diff
@@ -0,0 +1,2 @@
+> [!WARNING]
```

Comment:
Todo: Change this filename to `index_obsolete_core_[9.2,).partial.md` once 9.2 is released